x-pack/test/lists_api_integration/security_and_spaces/tests/update_exception_list_items.ts

```diff
@@ -31,6 +31,60 @@ export default ({ getService }: FtrProviderContext) => {
         await deleteAllExceptions(supertest, log);
       });
 
+      // SDH came up where this route was creating a new item when updating
+      // by item_id, ensuring that's no longer the case
+      it('should not create a new item on update', async () => {
```

Comment:
I think we could do away with the comment and put this context into the test description. I deleted the mention of an SDH because IMO the fact that it's a regression test isn't particularly important; deleting a test is an exception for which one had better have a good reason!

```suggestion
      it('should not create a new item when updating by item_id', async () => {
```